pull_request.py

```diff
@@ -235,6 +235,12 @@ def diff(self) -> unidiff.PatchSet:
             self._repo, self._pr_number))
         return unidiff.PatchSet(response.content.decode('utf-8'))
 
+    def creates_bonus(self) -> bool:
+        for added_file in self.diff().added_files:
+            if "/bonuses/" in added_file:
```

Comment:
I don't think this is sufficient to establish that a PR creates bonuses. The `/bonuses/` directory needs to be in a specific part of the file path. It also doesn't make clear that the PR _only_ creates bonuses, which is important for the application of most of the relevant rules. See `get_new_bonuses_or_raise` in this file.